language/fr/aboutpro.lang

```diff
@@ -1,7 +1,39 @@
 <?php
 
-include("{$_EnginePath}language/en/aboutpro.lang");
+$_Lang['DarkEnergyStatusIs']    = 'Énergie noire disponible ';
+$_Lang['BuyMoreDE']             = 'Acheter plus d\'énergie noire !';
```

Comment:
I've noticed a lot of translation strings which either end with a blank space (in the string), or with an exclamation or question marks prepended with a blank space. Is this something special related to the punctuation in the French language, or a mistake?

Reply:
Yes, French typography requires a white space before some signs of punctuation (: ; ! ?).

Reply:
Ok, thanks for the explanation.